packages/sonarwhal/src/lib/cli/wizards/shared-templates/package.hbs

```diff
@@ -68,7 +72,7 @@
     "lint": "npm-run-all lint:*",{{#if official}}
     "lint:js": "eslint . --cache --ext js --ext md --ext ts --ignore-path ../../.eslintignore --report-unused-disable-directives",{{else}}
     "lint:js": "eslint . --cache --ext js --ext md --ext ts --report-unused-disable-directives",{{/if}}
-    "lint:md": "markdownlint *.md",
+    "lint:md": "markdownlint --ignore CHANGELOG.md *.md",
```

Comment:
There are a few things more that need to be changed in this file:

* Add `"private": true`
* Remove `"author": ""`
* Add `"build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true",` if official
* Version should be `1.0.0`
* If official, `nyc` `extends` should be `../../.nycrc`

This applies to the parser's  `package.json` as well.

Reply:
done!